docs/products/postgresql/howto/use-dblink-extension.rst

```diff
@@ -0,0 +1,121 @@
+* ``TARGET_PG_DATABASE_NAME``: The remote database name
+
+.. Note::
+
+    If you're using Aiven for PostgreSQL as remote server, the above details are available in the `Aiven console <https://console.aiven.io/>`_ service Overview tab or via the dedicated ``avn service get`` command with the :ref:`Aiven CLI <avn_service_get>`.
+
+
+Enable ``dblink`` extension on Aiven for PostgreSQL
+-------------------------------------------------------
+
+To enable the ``dblink`` extension on an Aiven for PostgreSQL service:
+
+* Connect to the database with the ``avnadmin`` user, the following shows how to do it with ``psql``, the service URI can be found in the `Aiven console <https://console.aiven.io/>`_ service Overview page:
+
+::
+
+    psql "postgres://avnadmin:[AVNADMIN_PWD]@[PG_HOST]:[PG_PORT]/[PG_DB_NAME]?sslmode=require"
+
+.. Tip::
+
+    If you're using Aiven for PostgreSQL as remote server, you can connect to a service with the ``avnadmin`` user with the ``avn service cli`` command with the :ref:`Aiven CLI <avn-service-cli>`.
+
+* Create the ``dblink`` extension
+
+::
+
+    CREATE EXTENSION dblink;
+
+Create a foreign data wrapper using ``dblink_fdw``
+--------------------------------------------------
+
+To create a foreign data wrapper using the ``dblink_fwd`` you need to perform the following steps:
+
+*  Connect to the database with the ``avnadmin`` user, the following shows how to do it with ``psql``, the service URI can be found in the `Aiven console <https://console.aiven.io/>`_ service Overview page:
+
+::
+
+    psql "postgres://avnadmin:[AVNADMIN_PWD]@[PG_HOST]:[PG_PORT]/[PG_DB_NAME]?sslmode=require"
+
+
+* Create an user ``user1`` that will be access the ``dblink``
+
+::
+
+    CREATE USER user1 PASSWORD 'secret1'
+
+* Create a remote server definition (named ``pg_remote``)  using ``dblink_fdw`` and the target PostgreSQL connection details 
+
+::
+
+    CREATE SERVER pg_remote
+        FOREIGN DATA WRAPPER dblink_fdw
+        OPTIONS (
+                 host 'TARGET_PG_HOST',
+                 dbname 'TARGET_PG_DATABASE_NAME', 
+                 port 'TARGET_PG_PORT'
+                 );
+
+* Create a user mapping for the ``user1`` to automatically authenticate as the ``TARGET_PG_USER`` when using the ``dblink``
+
+::
+
+    CREATE USER MAPPING FOR user1
+        SERVER pg_remote
+        OPTIONS (
+            user 'TARGET_PG_USER', 
+            password 'TARGET_PG_PASSWORD'
+            );
+
+* Enable ``user1`` to use the remote PostgreSQL connection ``pg_remote``
+
+::
+
+    GRANT USAGE ON FOREIGN SERVER pg_remote TO user1;
+
+Query data using a foreign data wrapper
+---------------------------------------
+
+To query a foreign data wrapper you must be use a database user having the necessary grants to the remote server definition. In the previous example the user ``user1``. To query the remote table ``inventory`` defined in the target PostgreSQL database pointed by the ``pg_remote`` server definition:
```

Comment:
```suggestion
To query a foreign data wrapper you must be a database user having the necessary grants to the remote server definition. We can use ``user1`` from the previous example. To query the remote table ``inventory`` defined in the target PostgreSQL database pointed by the ``pg_remote`` server definition:
```
It's either "`be`" or "`use`" the database user - I've guessed which is wanted, but might have it wrong.

I think the short middle sentence needs reworking to make it read as a sentence - not sure if my suggestion is best.